src/hotspot/cpu/aarch64/aarch64.ad

```diff
@@ -16673,7 +16673,7 @@ instruct string_equalsL(iRegP_R1 str1, iRegP_R3 str2, iRegI_R4 cnt,
 
   format %{ "String Equals $str1,$str2,$cnt -> $result" %}
   ins_encode %{
-    // Count is in 8-bit bytes; non-Compact chars are 16 bits.
+    // Count is in 8-bit bytes; non-Compact chars are 8 bits.
```

Comment:
This change is a bit confusing: non-compact chars are still 16 bits, it's just at this point we know the string contains only 8-bit Latin characters. I think it's better to instead delete everything after the ";" (or leave it as it is).

Reply:
I have fixed this comment. Thank you for your suggestion.